docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.6"
+services:
+
+    webserver:
+      image: nginx:alpine
+      container_name: jsfprmvalidatorbundle-webserver
```

Comment:
I suggest to nor use container names as it's then impossible to run the same project second time (eg. two different branches on one machine)